test/SgidClient.spec.ts

```diff
@@ -56,26 +59,29 @@ describe('SgidClient', () => {
         clientId: MOCK_CLIENT_ID,
         clientSecret: MOCK_CLIENT_SECRET,
         privateKey: pkcs8Key,
-        redirectUri: MOCK_REDIRECT_URI,
+        redirectUris: [MOCK_REDIRECT_URI],
         hostname: MOCK_HOSTNAME,
-        apiVersion: MOCK_API_VERSION,
       })
 
       expect(pkcs8Client).toBeDefined()
     })
   })
 
-  describe('authorizationUrl', () => {
-    it('should generate authorisation URL correctly when only state is provided', () => {
+  describe('authorizationUrl (v2 with PKCE)', () => {
```

Comment:
minor point but I'm not sure it's helpful to add all this v2 naming if we're not keeping v1 anyway. it's a little confusing

Reply:
yup that makes sense! will make the change 👍